realis_e2e_test.go

```diff
@@ -864,7 +893,11 @@ func TestRealisClient_GetJobSummary(t *testing.T) {
 	summary, mErr := r.GetJobSummary(role)
 	assert.NoError(t, mErr)
 	assert.NotNil(t, summary)
-	assert.Equal(t, len(summary.Summaries), 1)
+	found := false
+	for _, s := range summary.Summaries {
+		found = found || (s.Job.TaskConfig.Job.Name == name)
```

Comment:
assert.True(t, found || (s.Job.TaskConfig.Job.Name == name))

Reply:
this is different. I am afraid that when we run this unit test, and there may be another job under role vagrant.
If we assertTrue for every job, it fails the unit test.
In this case, we just need to make sure the created job in this test is included in the list (summaries).

Reply:
sure. let me revert this change back